code/modules/awaymissions/corpse.dm

```diff
@@ -262,6 +299,146 @@
+	corpseid = 1
+	corpseidjob = "Clown"
+	corpseidaccess = "Clown"
+
+/obj/effect/landmark/corpse/mime
+	name = "Mime"
+	corpseuniform = /obj/item/clothing/under/mime
+	corpseshoes = /obj/item/clothing/shoes/black
+	corpseradio = /obj/item/device/radio/headset
+	corpsemask = /obj/item/clothing/mask/gas/mime
+	corpsegloves = /obj/item/clothing/gloves/white/stunglove // Spawn with empty, crappy batteries.
+	corpseback = /obj/item/weapon/storage/backpack
+	corpseid = 1
+	corpseidjob = "Mime"
+	corpseidaccess = "Mime"
+
+/obj/effect/landmark/corpse/janitor
+	name = "Janitor"
+	corpseradio = /obj/item/device/radio/headset/headset_cargo
+	corpseuniform = /obj/item/clothing/under/rank/janitor
+	corpseshoes = /obj/item/clothing/shoes/black
+	corpseback = /obj/item/weapon/storage/backpack
+	corpsebelt = /obj/item/weapon/storage/belt/janitor
+	corpsegloves = /obj/item/clothing/gloves/purple
+	corpsehelmet = /obj/item/clothing/head/soft/purple
+	corpseid = 1
+	corpseidjob = "Janitor"
+	corpseidaccess = "Janitor"
+
+/obj/effect/landmark/corpse/janitor/chempack
+	corpseback = /obj/item/weapon/reagent_containers/chempack
+	corpseglasses = /obj/item/clothing/glasses/sunglasses
+
+/obj/effect/landmark/corpse/librarian
+	name = "Librarian"
+	corpseuniform = /obj/item/clothing/under/suit_jacket/red
+	corpseshoes = /obj/item/clothing/shoes/black
+	corpseback = /obj/item/weapon/storage/backpack
+	corpseradio = /obj/item/device/radio/headset
+	corpseid = 1
+	corpseidjob = "Librarian"
+	corpseidaccess = "Librarian"
+
+/obj/effect/landmark/corpse/internalaffairsagent
+	name = "Internal Affairs Agent"
+	corpseuniform = /obj/item/clothing/under/rank/internalaffairs
+	corpsesuit = /obj/item/clothing/suit/storage/internalaffairs
+	corpseshoes = /obj/item/clothing/shoes/centcom
+	corpseback = /obj/item/weapon/storage/backpack
+	corpseradio = /obj/item/device/radio/headset
+	corpseglasses = /obj/item/clothing/glasses/sunglasses
+	corpseid = 1
+	corpseidjob = "Internal Affairs Agent"
+	corpseidaccess = "Internal Affairs Agent"
+
+/obj/effect/landmark/corpse/internalaffairsagent/lawyer
+	name = "Lawyer"
+	corpseuniform = /obj/item/clothing/under/lawyer/bluesuit
+	corpsesuit = /obj/item/clothing/suit/storage/lawyer/bluejacket
+	corpseshoes = /obj/item/clothing/shoes/leather
+	corpseidjob = "Lawyer"
+
+/obj/effect/landmark/corpse/internalaffairsagent/bridgeofficer
+	name = "Bridge Officer"
+	corpseuniform = /obj/item/clothing/under/bridgeofficer
+	corpsesuit = /obj/item/clothing/suit/storage/lawyer/bridgeofficer
+	corpsegloves = /obj/item/clothing/gloves/white
+	corpsehelmet = /obj/item/clothing/head/soft/bridgeofficer
+	corpseidjob = "Bridge Officer"
+
+/obj/effect/landmark/corpse/chaplain
+	name = "Chaplain"
+	corpseuniform = /obj/item/clothing/under/rank/chaplain
+	corpseback = /obj/item/weapon/storage/backpack
+	corpseradio = /obj/item/device/radio/headset
+	corpseshoes = /obj/item/clothing/shoes/laceup
+	corpseid = 1
+	corpseidjob = "Captain"
```

Comment:
is the chaplain meant to have cap access

Reply:
definitely not